sdk/src/main/scala/com/horizen/block/MainchainBlockReference.scala

```diff
@@ -162,7 +163,8 @@ object MainchainBlockReference {
   }
 
   // Try to parse Mainchain block and return MainchainHeader, SCMap and MainchainTransactions sequence.
-  private def parseMainchainBlockBytes(mainchainBlockBytes: Array[Byte]): Try[(MainchainHeader, Seq[MainchainTransaction])] = Try {
+  private def parseMainchainBlockBytes(mainchainBlockBytes: Array[Byte]):
```

Comment:
Update MainchainBlockReference parsing Unit tests with:
1. MainchainBlock of new version with no certificates inside.
2. Mainchain block of new version with a single certificate, that belongs to our SC.
3. Mainchain block of new version with a single certificate, that belongs to different SC.
4. Mainchain block of new version with a multiple certificate, that belongs to several Sidechains, including our SC.
5. Mainchain block of new version with both transactions and certificates

Reply:
6. MainchainBlocks of new version with certificate, forward transfers and sidechain creations in different combinations. Test should cover cases in SCMap calculation changes and semantic validity changes.
7. Probably update old tests data with new versioned blocks.

Reply:
Done.